### ISSUE TITLE
Underlords hard crashing macOS at launch

### ISSUE THREAD

**Issue by akonyha** [ValveSoftware/Dota-Underlords#44]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/): 
https://gist.github.com/akonyha/30b7cdb58026af132d992360cf42c0fb
* Have you checked for system updates?: [Yes/No]
Yes
* Are you using the latest stable video driver available for your system? [Yes/No]
Yes
* Have you verified the game files?: [Yes/No]
Yes

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large pastes as a [Github Gist](https://gist.github.com/).

Verified running macOS 10.15.4 with a Metal-capable GPU. Copied below is the Graphics/Display info: 

  Chipset Model:	Intel Iris Plus Graphics
  Type:	GPU
  Bus:	Built-In
  VRAM (Dynamic, Max):	1536 MB
  Vendor:	Intel
  Device ID:	0x8a51
  Revision ID:	0x0007
  Metal:	Supported, feature set macOS GPUFamily2 v1
  Displays:
Color LCD:
  Display Type:	Built-In Retina LCD
  Resolution:	2560 x 1600 Retina
  Framebuffer Depth:	30-Bit Color (ARGB2101010)
  Main Display:	Yes
  Mirror:	Off
  Online:	Yes
  Automatically Adjust Brightness:	No
  Connection Type:	Internal

Basically when I launch Underlords, I get a black screen with a cursor, the cursor turns into the "beach ball" for a few seconds, the back to the normal cursor for a few seconds that is now unresponsive. After a few seconds my MBA hard crashes and restarts.

#### Steps for reproducing this issue:

1. Launch Underlords
2. MBA will hang after beachball and entire OS will crash

**Comment by akonyha** [comment]
<details><summary>Further here are logs from the hard crash:</summary><p>

```
panic(cpu 2 caller 0xffffff800824c32a): Kernel trap at 0xffffff7f8c2e4b25, type 14=page fault, registers:
CR0: 0x0000000080010033, CR2: 0x0000000000000000, CR3: 0x00000003d9aca1b9, CR4: 0x00000000003626e0
RAX: 0x0000000000000000, RBX: 0x0000000000000002, RCX: 0x0000000000000000, RDX: 0xc600000020014000
RSP: 0xffffff921a2fb350, RBP: 0xffffff921a2fb5a0, RSI: 0x00000000fffffffc, RDI: 0x0000000000000000
R8:  0x0000000000000000, R9:  0x0000000000000000, R10: 0x0000000000000001, R11: 0x0000000000000041
R12: 0xffffff9221a0a85c, R13: 0xffffff803cf44000, R14: 0xffffff921f3fb0c0, R15: 0xffffff804e991ba0
RFL: 0x0000000000010246, RIP: 0xffffff7f8c2e4b25, CS:  0x0000000000000008, SS:  0x0000000000000010
Fault CR2: 0x0000000000000000, Error code: 0x0000000000000000, Fault CPU: 0x2, PL: 0, VF: 0

Backtrace (CPU 2), Frame : Return Address
0xffffff921a2fadb0 : 0xffffff80081215cd 
0xffffff921a2fae00 : 0xffffff800825a3c5 
0xffffff921a2fae40 : 0xffffff800824bf7e 
0xffffff921a2fae90 : 0xffffff80080c7a40 
0xffffff921a2faeb0 : 0xffffff8008120c97 
0xffffff921a2fafb0 : 0xffffff8008121087 
0xffffff921a2fb000 : 0xffffff80088c2c7c 
0xffffff921a2fb070 : 0xffffff800824c32a 
0xffffff921a2fb1f0 : 0xffffff800824c028 
0xffffff921a2fb240 : 0xffffff80080c7a40 
0xffffff921a2fb260 : 0xffffff7f8c2e4b25 
0xffffff921a2fb5a0 : 0xffffff7f8c2babe1 
0xffffff921a2fb5e0 : 0xffffff7f8be42d53 
0xffffff921a2fb610 : 0xffffff7f8c2ba7e4 
0xffffff921a2fb650 : 0xffffff7f8be42ee3 
0xffffff921a2fb690 : 0xffffff7f8be434ed 
0xffffff921a2fb930 : 0xffffff7f8c2bb180 
0xffffff921a2fb960 : 0xffffff7f8be44429 
0xffffff921a2fb9f0 : 0xffffff7f8be425b0 
0xffffff921a2fba50 : 0xffffff7f8be4239d 
0xffffff921a2fbaa0 : 0xffffff7f8be41467 
0xffffff921a2fbae0 : 0xffffff8008854f3e 
0xffffff921a2fbb30 : 0xffffff7f8c2badea 
0xffffff921a2fbb60 : 0xffffff800885e163 
0xffffff921a2fbca0 : 0xffffff8008209bb2 
0xffffff921a2fbdb0 : 0xffffff8008127328 
0xffffff921a2fbe10 : 0xffffff80080fdcc5 
0xffffff921a2fbe70 : 0xffffff8008114aa2 
0xffffff921a2fbf00 : 0xffffff8008232605 
0xffffff921a2fbfa0 : 0xffffff80080c8226 
      Kernel Extensions in backtrace:
         com.apple.iokit.IOAcceleratorFamily2(438.4.5)[33DA860E-90A6-3224-A944-93FA125B054E]@0xffffff7f8bdff000->0xffffff7f8bec2fff
            dependency: com.apple.driver.AppleMobileFileIntegrity(1.0.5)[5EF51D44-9FC7-357F-9FB7-3468FD748382]@0xffffff7f8922e000
            dependency: com.apple.iokit.IOPCIFamily(2.9)[1B1F3BBB-9212-3CF9-94F8-8FEF0D3ACEC4]@0xffffff7f88b11000
            dependency: com.apple.iokit.IOSurface(269.11)[E71EAC82-A838-3430-B5D1-2CB347E6CF59]@0xffffff7f8a678000
            dependency: com.apple.iokit.IOGraphicsFamily(575.1)[D47CA481-C5E5-3F03-9B04-6634DF5F3121]@0xffffff7f8bd91000
            dependency: com.apple.iokit.IOReportFamily(47)[337B24B2-0261-3521-8D1B-23387FC1D657]@0xffffff7f88a49000
         com.apple.driver.AppleIntelICLGraphics(14.0.5)[7EA86F7B-5D84-354B-B0AC-1009876EDE53]@0xffffff7f8c28d000->0xffffff7f8c3cafff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[1B1F3BBB-9212-3CF9-94F8-8FEF0D3ACEC4]@0xffffff7f88b11000
            dependency: com.apple.iokit.IOSurface(269.11)[E71EAC82-A838-3430-B5D1-2CB347E6CF59]@0xffffff7f8a678000
            dependency: com.apple.iokit.IOGraphicsFamily(575.1)[D47CA481-C5E5-3F03-9B04-6634DF5F3121]@0xffffff7f8bd91000
            dependency: com.apple.iokit.IOAcceleratorFamily2(438.4.5)[33DA860E-90A6-3224-A944-93FA125B054E]@0xffffff7f8bdff000

BSD process name corresponding to current thread: underlords
Boot args: chunklist-security-epoch=0 -chunklist-no-rev2-dev

Mac OS version:
19E287

Kernel version:
Darwin Kernel Version 19.4.0: Wed Mar  4 22:28:40 PST 2020; root:xnu-6153.101.6~15/RELEASE_X86_64
Kernel UUID: AB0AA7EE-3D03-3C21-91AD-5719D79D7AF6
Kernel slide:     0x0000000007e00000
Kernel text base: 0xffffff8008000000
__HIB  text base: 0xffffff8007f00000
System model name: MacBookAir9,1 (Mac-0CFF9C7C2B63DF8D)
System shutdown begun: NO

System uptime in nanoseconds: 97150429255790
last loaded kext at 84673936505378: >usb.cdc.acm	5.0.0 (addr 0xffffff7f8c216000, size 32768)
last unloaded kext at 84737916253114: >!UMergeNub	900.4.2 (addr 0xffffff7f8c138000, size 12288)
loaded kexts:
>!AGraphicsDevicePolicy	5.1.16
@fileutil	20.036.15
@AGDCPluginDisplayMetrics	5.1.16
>!AHV	1
|IOUserEthernet	1.0.1
|IO!BSerialManager	7.0.4f6
>pmtelemetry	1
>AGPM	111.4.4
>X86PlatformShim	1.0.0
>!AUpstreamUserClient	3.6.8
>!APlatformEnabler	2.7.0d0
@Dont_Steal_Mac_OS_X	7.0.0
>!A!IICLGraphics	14.0.5
>BridgeAudioCommunication	6.70.3
>!AThunderboltIP	3.1.4
>!ABacklight	180.3
>!A!ISlowAdaptiveClocking	4.0.0
>!AAVEBridge	6.1
>!ABridgeAudio!C	6.70.3
>!ATopCaseHIDEventDriver	3430.1
>!AGFXHDA	100.1.426
>!A!IPCHPMC	2.0.1
>!AHIDALSService	1
>!A!IICLLPGraphicsFramebuffer	14.0.5
>!AMCCSControl	1.11
@filesystems.autofs	3.0
@filesystems.apfs	1412.101.1
>BCMWLANFirmware4355.Hashstore	1
>BCMWLANFirmware4364.Hashstore	1
>BCMWLANFirmware4377.Hashstore	1
>!A!BModule	1
@filesystems.hfs.kext	522.100.5
@BootCache	40
@!AFSCompression.!AFSCompressionTypeDataless	1.0.0d1
@!AFSCompression.!AFSCompressionTypeZlib	1.0.0
>!AVirtIO	1.0
@private.KextAudit	1.0
>!ASmartBatteryManager	161.0.0
>!ABCMWLANBusInterfacePCIe	1
>!AACPIButtons	6.1
>!ASMBIOS	2.1
>!AACPIEC	6.1
>!AAPIC	1.7
$!AImage4	1
@nke.applicationfirewall	303
$TMSafetyNet	8
@!ASystemPolicy	2.0.0
|EndpointSecurity	1
>!AGraphicsControl	5.1.16
|IOAVB!F	840.3
>!ASSE	1.0
@!AGPUWrangler	5.1.16
>!ABacklightExpert	1.1.0
|IOSlowAdaptiveClocking!F	1.0.0
>X86PlatformPlugin	1.0.0
>!AActuatorDriver	3440.1
>!AHS!BDriver	3430.1
>IO!BHIDDriver	7.0.4f6
>!AMultitouchDriver	3440.1
>!AInputDeviceSupport	3440.8
|IONDRVSupport	575.1
>IOPlatformPlugin!F	6.0.0d8
>!AHIDKeyboard	209
@!AGraphicsDeviceControl	5.1.16
|IOAccelerator!F2	438.4.5
>!ASMBus!C	1.0.18d1
|IOGraphics!F	575.1
@plugin.IOgPTPPlugin	840.3
|IOEthernetAVB!C	1.1.0
@kext.triggers	1.0
>usb.cdc.ncm	5.0.0
>usb.cdc	5.0.0
>usb.networking	5.0.0
>usb.!UHostCompositeDevice	1.2
>usb.!UVHCIBCE	1.2
>usb.!UVHCI	1.2
>usb.!UVHCICommonBCE	1.0
>usb.!UVHCICommon	1.0
>!AEffaceableNOR	1.0
|IOBufferCopy!C	1.1.0
|IOBufferCopyEngine!F	1
|IONVMe!F	2.1.0
>IO!BHost!CPCIeTransport	7.0.4f6
|IO!BHost!CTransport	7.0.4f6
>!A!BDebug	1
>!AConvergedIPCOLYBTControl	1
>!AConvergedPCI	1
>!A!BDebugService	1
>!AThunderboltPCIDownAdapter	2.5.4
>!AThunderboltDPInAdapter	6.2.6
>!AThunderboltDPAdapter!F	6.2.6
>!AHPM	3.4.4
>!A!ILpssI2C!C	3.0.60
>!A!ILpssDmac	3.0.60
>!A!ILpssI2C	3.0.60
>!AThunderboltNHI	5.8.6
|IOThunderbolt!F	7.6.0
>usb.!UXHCIPCI	1.2
>usb.!UXHCI	1.2
|IOSurface	269.11
@filesystems.hfs.encodings.kext	1
|IOAudio!F	300.2
@vecLib.kext	1.2.0
|IOUSB!F	900.4.2
>!AMultiFunctionManager	1
>!AEFINVRAM	2.1
>!AEFIRuntime	2.1
>!ABCMWLANCore	1.0.0
>mDNSOffloadUserClient	1.0.1b8
>IOImageLoader	1.0.0
|IOSerial!F	11
|IO80211!FV2	1200.12.2b1
>corecapture	1.0.4
|IOSkywalk!F	1
>!ASMCRTC	1.0
|IOSMBus!F	1.1
|IOHID!F	2.0.0
$quarantine	4
$sandbox	300.0
@kext.!AMatch	1.0.0d1
>!AKeyStore	2
>!UTDM	489.101.1
|IOSCSIBlockCommandsDevice	422.101.1
>!ACredentialManager	1.0
>KernelRelayHost	1
>!ASEPManager	1.0.1
>IOSlaveProcessor	1
>!AFDEKeyStore	28.30
>!AEffaceable!S	1.0
>!AMobileFileIntegrity	1.0.5
@kext.CoreTrust	1
|CoreAnalytics!F	1
|IOTimeSync!F	840.3
|IONetworking!F	3.4
>DiskImages	493.0.0
|IO!B!F	7.0.4f6
|IO!BPacketLogger	7.0.4f6
|IOUSBMass!SDriver	157.101.3
|IOSCSIArchitectureModel!F	422.101.1
|IO!S!F	2.1
|IOUSBHost!F	1.2
>usb.!UCommon	1.0
>!UHostMergeProperties	1.2
>!ABusPower!C	1.0
|IOReport!F	47
>!AACPIPlatform	6.1
>!ASMC	3.1.9
>watchdog	1
|IOPCI!F	2.9
|IOACPI!F	1.4
@kec.pthread	1
@kec.corecrypto	1.0
@kec.Libm	1
```

</p></details>

**Comment by chriszo111** [comment]
I'm curious whether this is related to a config issue. Can you paste your `common\Underlords\game\dac\cfg\video.txt`?
